### ISSUE TITLE
Installation of paddle2.3.2 error

### ISSUE THREAD

**Issue by rojinakashefi** [PaddlePaddle/Paddle#59160]
### 问题描述 Issue Description

I am using google colab environment
trying to install paddle 2.3.2 for python3.10 form this [link](https://www.paddlepaddle.org.cn/install/old?docurl=/documentation/docs/zh/install/conda/linux-conda.html#old-version-anchor-10-V2.3) cpu version. however when i import paddle i get this error: Error: Can not import avx core while this file exists: /usr/local/lib/python3.10/dist-packages/paddle/fluid/core_avx.so

### 版本&环境信息 Version & Environment Information

Paddle version: N/A
Paddle With CUDA: N/A

OS: ubuntu 22.04
GCC version: (Ubuntu 11.4.0-1ubuntu1~22.04) 11.4.0
Clang version: 14.0.0-1ubuntu1.1
CMake version: version 3.27.7
Libc version: glibc 2.35
Python version: 3.10.12

CUDA version: 11.8.89
Build cuda_11.8.r11.8/compiler.31833905_0
cuDNN version: 8.9.6
Nvidia driver version: N/A
Nvidia driver List: N/A

**Comment by zyfncg** [comment]
Is there more error info like following?
```
Error: Can not import avx core while this file exists: /usr/local/python3/lib/python3.8/site-packages/paddle/fluid/core_avx.so
Traceback (most recent call last):
File "test.py", line 1, in
import paddle
File "/usr/local/python3/lib/python3.8/site-packages/paddle/__init__.py", line 25, in
from .fluid import monkey_patch_variable
File "/usr/local/python3/lib/python3.8/site-packages/paddle/fluid/__init__.py", line 36, in
from . import framework
File "/usr/local/python3/lib/python3.8/site-packages/paddle/fluid/framework.py", line 37, in
from . import core
File "/usr/local/python3/lib/python3.8/site-packages/paddle/fluid/core.py", line 297, in
raise e
File "/usr/local/python3/lib/python3.8/site-packages/paddle/fluid/core.py", line 257, in
from .core_avx import *
ImportError: /lib64/libstdc++.so.6: version `CXXABI_1.3.8' not found (required by /usr/local/python3/lib/python3.8/site-packages/paddle/fluid/core_avx.so)
```

**Comment by doylecl** [comment]
![image](https://github.com/PaddlePaddle/Paddle/assets/55191338/48d8b884-36c6-415c-8326-5bd2d776753e)
你好，问一下，这种应该怎么处理，docker中 pip下载不成功

**Comment by zyfncg** [comment]
<img width="1297" alt="image" src="https://github.com/PaddlePaddle/Paddle/assets/13048366/83a7d053-0759-4f7d-bbab-47fe618f4dc6">
换下载源试试？

**Comment by rojinakashefi** [comment]
> Is there more error info like following?
> 
> ```
> Error: Can not import avx core while this file exists: /usr/local/python3/lib/python3.8/site-packages/paddle/fluid/core_avx.so
> Traceback (most recent call last):
> File "test.py", line 1, in
> import paddle
> File "/usr/local/python3/lib/python3.8/site-packages/paddle/__init__.py", line 25, in
> from .fluid import monkey_patch_variable
> File "/usr/local/python3/lib/python3.8/site-packages/paddle/fluid/__init__.py", line 36, in
> from . import framework
> File "/usr/local/python3/lib/python3.8/site-packages/paddle/fluid/framework.py", line 37, in
> from . import core
> File "/usr/local/python3/lib/python3.8/site-packages/paddle/fluid/core.py", line 297, in
> raise e
> File "/usr/local/python3/lib/python3.8/site-packages/paddle/fluid/core.py", line 257, in
> from .core_avx import *
> ImportError: /lib64/libstdc++.so.6: version `CXXABI_1.3.8' not found (required by /usr/local/python3/lib/python3.8/site-packages/paddle/fluid/core_avx.so)
> ```

First I run this:
`python -m pip install paddlepaddle==2.3.2 -i https://pypi.tuna.tsinghua.edu.cn/simple`
The output:
`Looking in indexes: https://pypi.tuna.tsinghua.edu.cn/simple
Collecting paddlepaddle==2.3.2
  Downloading https://pypi.tuna.tsinghua.edu.cn/packages/36/04/50d7285b5ee10566e45b01e598b140e2a1d351392ec2a7f1361d78896b0e/paddlepaddle-2.3.2-cp310-cp310-manylinux1_x86_64.whl (112.6 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 112.6/112.6 MB 5.0 MB/s eta 0:00:00
Requirement already satisfied: requests>=2.20.0 in /usr/local/lib/python3.10/dist-packages (from paddlepaddle==2.3.2) (2.31.0)
Requirement already satisfied: numpy>=1.13 in /usr/local/lib/python3.10/dist-packages (from paddlepaddle==2.3.2) (1.23.5)
Collecting protobuf<=3.20.0,>=3.1.0 (from paddlepaddle==2.3.2)
  Downloading https://pypi.tuna.tsinghua.edu.cn/packages/5c/a6/9a451d69492a695e2300643933590762aa4e5bbef2ae5f08f7d218ea6c3a/protobuf-3.20.0-cp310-cp310-manylinux_2_12_x86_64.manylinux2010_x86_64.whl (1.1 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 1.1/1.1 MB 10.5 MB/s eta 0:00:00
Requirement already satisfied: Pillow in /usr/local/lib/python3.10/dist-packages (from paddlepaddle==2.3.2) (9.4.0)
Requirement already satisfied: six in /usr/local/lib/python3.10/dist-packages (from paddlepaddle==2.3.2) (1.16.0)
Requirement already satisfied: decorator in /usr/local/lib/python3.10/dist-packages (from paddlepaddle==2.3.2) (4.4.2)
Collecting astor (from paddlepaddle==2.3.2)
  Downloading https://pypi.tuna.tsinghua.edu.cn/packages/c3/88/97eef84f48fa04fbd6750e62dcceafba6c63c81b7ac1420856c8dcc0a3f9/astor-0.8.1-py2.py3-none-any.whl (27 kB)
Collecting paddle-bfloat==0.1.7 (from paddlepaddle==2.3.2)
  Downloading https://pypi.tuna.tsinghua.edu.cn/packages/72/cc/4e6149a9a94f0e1449686f2e5152b734bbc7ddce18ea3b240995d39fc642/paddle_bfloat-0.1.7-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (383 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 383.2/383.2 kB 16.0 MB/s eta 0:00:00
Requirement already satisfied: opt-einsum==3.3.0 in /usr/local/lib/python3.10/dist-packages (from paddlepaddle==2.3.2) (3.3.0)
Requirement already satisfied: charset-normalizer<4,>=2 in /usr/local/lib/python3.10/dist-packages (from requests>=2.20.0->paddlepaddle==2.3.2) (3.3.2)
Requirement already satisfied: idna<4,>=2.5 in /usr/local/lib/python3.10/dist-packages (from requests>=2.20.0->paddlepaddle==2.3.2) (3.4)
Requirement already satisfied: urllib3<3,>=1.21.1 in /usr/local/lib/python3.10/dist-packages (from requests>=2.20.0->paddlepaddle==2.3.2) (2.0.7)
Requirement already satisfied: certifi>=2017.4.17 in /usr/local/lib/python3.10/dist-packages (from requests>=2.20.0->paddlepaddle==2.3.2) (2023.7.22)
Installing collected packages: paddle-bfloat, protobuf, astor, paddlepaddle
  Attempting uninstall: protobuf
    Found existing installation: protobuf 3.20.3
    Uninstalling protobuf-3.20.3:
      Successfully uninstalled protobuf-3.20.3
ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
google-api-core 2.11.1 requires protobuf!=3.20.0,!=3.20.1,!=4.21.0,!=4.21.1,!=4.21.2,!=4.21.3,!=4.21.4,!=4.21.5,<5.0.0.dev0,>=3.19.5, but you have protobuf 3.20.0 which is incompatible.
google-cloud-bigquery 3.12.0 requires protobuf!=3.20.0,!=3.20.1,!=4.21.0,!=4.21.1,!=4.21.2,!=4.21.3,!=4.21.4,!=4.21.5,<5.0.0dev,>=3.19.5, but you have protobuf 3.20.0 which is incompatible.
google-cloud-bigquery-connection 1.12.1 requires protobuf!=3.20.0,!=3.20.1,!=4.21.0,!=4.21.1,!=4.21.2,!=4.21.3,!=4.21.4,!=4.21.5,<5.0.0dev,>=3.19.5, but you have protobuf 3.20.0 which is incompatible.
google-cloud-bigquery-storage 2.22.0 requires protobuf!=3.20.0,!=3.20.1,!=4.21.0,!=4.21.1,!=4.21.2,!=4.21.3,!=4.21.4,!=4.21.5,<5.0.0dev,>=3.19.5, but you have protobuf 3.20.0 which is incompatible.
google-cloud-datastore 2.15.2 requires protobuf!=3.20.0,!=3.20.1,!=4.21.0,!=4.21.1,!=4.21.2,!=4.21.3,!=4.21.4,!=4.21.5,<5.0.0dev,>=3.19.5, but you have protobuf 3.20.0 which is incompatible.
google-cloud-firestore 2.11.1 requires protobuf!=3.20.0,!=3.20.1,!=4.21.0,!=4.21.1,!=4.21.2,!=4.21.3,!=4.21.4,!=4.21.5,<5.0.0dev,>=3.19.5, but you have protobuf 3.20.0 which is incompatible.
google-cloud-functions 1.13.3 requires protobuf!=3.20.0,!=3.20.1,!=4.21.0,!=4.21.1,!=4.21.2,!=4.21.3,!=4.21.4,!=4.21.5,<5.0.0dev,>=3.19.5, but you have protobuf 3.20.0 which is incompatible.
google-cloud-iam 2.12.2 requires protobuf!=3.20.0,!=3.20.1,!=4.21.0,!=4.21.1,!=4.21.2,!=4.21.3,!=4.21.4,!=4.21.5,<5.0.0dev,>=3.19.5, but you have protobuf 3.20.0 which is incompatible.
google-cloud-language 2.9.1 requires protobuf!=3.20.0,!=3.20.1,!=4.21.0,!=4.21.1,!=4.21.2,!=4.21.3,!=4.21.4,!=4.21.5,<5.0.0dev,>=3.19.5, but you have protobuf 3.20.0 which is incompatible.
google-cloud-resource-manager 1.10.4 requires protobuf!=3.20.0,!=3.20.1,!=4.21.0,!=4.21.1,!=4.21.2,!=4.21.3,!=4.21.4,!=4.21.5,<5.0.0dev,>=3.19.5, but you have protobuf 3.20.0 which is incompatible.
google-cloud-translate 3.11.3 requires protobuf!=3.20.0,!=3.20.1,!=4.21.0,!=4.21.1,!=4.21.2,!=4.21.3,!=4.21.4,!=4.21.5,<5.0.0dev,>=3.19.5, but you have protobuf 3.20.0 which is incompatible.
googleapis-common-protos 1.61.0 requires protobuf!=3.20.0,!=3.20.1,!=4.21.1,!=4.21.2,!=4.21.3,!=4.21.4,!=4.21.5,<5.0.0.dev0,>=3.19.5, but you have protobuf 3.20.0 which is incompatible.
grpc-google-iam-v1 0.12.7 requires protobuf!=3.20.0,!=3.20.1,!=4.21.1,!=4.21.2,!=4.21.3,!=4.21.4,!=4.21.5,<5.0.0dev,>=3.19.5, but you have protobuf 3.20.0 which is incompatible.
tensorflow 2.14.0 requires protobuf!=4.21.0,!=4.21.1,!=4.21.2,!=4.21.3,!=4.21.4,!=4.21.5,<5.0.0dev,>=3.20.3, but you have protobuf 3.20.0 which is incompatible.
tensorflow-metadata 1.14.0 requires protobuf<4.21,>=3.20.3, but you have protobuf 3.20.0 which is incompatible.
Successfully installed astor-0.8.1 paddle-bfloat-0.1.7 paddlepaddle-2.3.2 protobuf-3.20.0`

Then I check pip:
`pip show paddlepaddle`
The ouput
`Name: paddlepaddle
Version: 2.3.2
Summary: Parallel Distributed Deep Learning
Home-page: 
Author: 
Author-email: [Paddle-better@baidu.com](mailto:Paddle-better@baidu.com)
License: Apache Software License
Location: /usr/local/lib/python3.10/dist-packages
Requires: astor, decorator, numpy, opt-einsum, paddle-bfloat, Pillow, protobuf, requests, six
Required-by:`

which show that paddlepaddle successfully installed.

Then i run
`import paddle`
the output:
`Error: Can not import avx core while this file exists: /usr/local/lib/python3.10/dist-packages/paddle/fluid/core_avx.so
---------------------------------------------------------------------------
ImportError                               Traceback (most recent call last)
[<ipython-input-4-5b4954d53387>](https://localhost:8080/#) in <cell line: 1>()
----> 1 import paddle

6 frames
[/usr/local/lib/python3.10/dist-packages/paddle/__init__.py](https://localhost:8080/#) in <module>
     23 
     24 from .batch import batch  # noqa: F401
---> 25 from .framework import monkey_patch_variable
     26 from .framework import monkey_patch_math_varbase
     27 monkey_patch_variable()

[/usr/local/lib/python3.10/dist-packages/paddle/framework/__init__.py](https://localhost:8080/#) in <module>
     15 # TODO: import framework api under this directory
     16 
---> 17 from . import random  # noqa: F401
     18 from .random import seed  # noqa: F401
     19 from .framework import get_default_dtype  # noqa: F401

[/usr/local/lib/python3.10/dist-packages/paddle/framework/random.py](https://localhost:8080/#) in <module>
     14 
     15 # TODO: define random api
---> 16 import paddle.fluid as fluid
     17 from paddle.fluid import core
     18 

[/usr/local/lib/python3.10/dist-packages/paddle/fluid/__init__.py](https://localhost:8080/#) in <module>
     34 
     35 # import all class inside framework into fluid module
---> 36 from . import framework
     37 from .framework import *
     38 # import all class inside executor into fluid module

[/usr/local/lib/python3.10/dist-packages/paddle/fluid/framework.py](https://localhost:8080/#) in <module>
     35 from .proto import framework_pb2
     36 
---> 37 from . import core
     38 from . import unique_name
     39 import paddle.version as fluid_version

[/usr/local/lib/python3.10/dist-packages/paddle/fluid/core.py](https://localhost:8080/#) in <module>
    296                 'Error: Can not import avx core while this file exists: ' +
    297                 current_path + os.sep + 'core_avx.' + core_suffix + '\n')
--> 298             raise e
    299         else:
    300             from .. import compat as cpt

[/usr/local/lib/python3.10/dist-packages/paddle/fluid/core.py](https://localhost:8080/#) in <module>
    254 if avx_supported():
    255     try:
--> 256         from . import core_avx
    257         core_avx.LoDTensor = core_avx.Tensor
    258 

ImportError: /usr/local/lib/python3.10/dist-packages/paddle/fluid/core_avx.so: undefined symbol: _dl_sym, version GLIBC_PRIVATE

---------------------------------------------------------------------------
NOTE: If your import is failing due to a missing package, you can
manually install dependencies using either !pip or !apt.

To view examples of installing some common dependencies, click the
"Open Examples" button below.
---------------------------------------------------------------------------`

I have to mention i have to install paddle=2.3.2.

**Comment by tianshuo78520a** [comment]
There are indeed issues with using Paddle2.3.2 in Ubuntu 22.04, and we have fixed the issue in version 2.4

Solution:
1. Install Paddle version 2.4.2 or 2.5.2
2. Using ubuntu20.04 version with paddlepaddle=2.3.2 can successfully import paddles as normal

2.3.2 is an older version, we recommend using a new version of the pad
